charts/datadog/templates/agent-rbac.yaml

```diff
@@ -22,6 +22,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
This should be in DCA RBAC no? And it should be protected by a `{{ if compliance }}`.

BTW Following latest custom checks, we need this extra set of permissions in DCA

```
- apiGroups:
  - ""
  resources:
  - serviceaccounts
  - namespaces
  verbs:
  - list
- apiGroups:
  - rbac.authorization.k8s.io
  resources:
  - clusterrolebindings
  - rolebindings
  verbs:
  - list
- apiGroups:
  - networking.k8s.io
  resources:
  - networkpolicies
  verbs:
  - list
```

Reply:
> This should be in DCA RBAC no

Confused, this is indeed for DCA, see up top -- 
```
{{- if and .Values.clusterAgent.enabled .Values.clusterAgent.rbac.create -}}
```
https://github.com/DataDog/helm-charts/blob/e4f7fd1503244ffab848d1378ea0e814deaf5851/charts/datadog/templates/agent-rbac.yaml#L1

Maybe we should rename this file (misleading).

Reply:
Updated RBAC after rebase